public/scripts/homepage.js

```diff
@@ -0,0 +1,29 @@
+function showAnswer() {
+    var ele = document.getElementById("answer");
+    ele.classList.remove("hidden");
+    ele.style.display = "block";
+}
+
+// function submitFeedback() {
+//     document.getElementById("nextQuestion").submit();
+// }
+
+function myFunction() {
+    var x = document.getElementById("opt").selectedIndex;
+    var options = document.getElementsByTagName("option")[x].value;
+    var fileName = options.slice(0, -3);
+    // alert(url)
+    location.href = "http://localhost:4000/flash/" + fileName + "/submitFeedback";
```

Comment:
how will it work when we deploy to cloud?

Reply:
ok i can change this

Reply:
can you check now everything works perfectly

Reply:
if any changes required please update me will do it